src/__tests__/OPLS.test.js

```diff
@@ -412,20 +412,20 @@ describe('prediction with metadata', () => {
   });
 
   it('test prediction auc', () => {
-    expect(prediction.auc).toBeCloseTo(0.55826, 4);
+    expect(prediction.auc).toBeCloseTo(0.9989, 4);
   });
 
   it('test prediction tPred vector', () => {
     expect(prediction.tPred.to1DArray()).toBeDeepCloseTo(
       model.getLogs().tPred.to1DArray(),
-      5,
+      8,
     );
   });
 
   it('test prediction tOrth vector', () => {
     expect(prediction.tOrth.to1DArray()).toBeDeepCloseTo(
       model.getLogs().tOrth.to1DArray(),
-      5,
+      8,
     );
   });
```

Comment:
@jwist The accuracy of scores prediction over the same data set used to build the model increased with this modifications (when Y is an array of categories).

Reply:
the if the accuracy increases you should be able to change the level of digits to 5 or 6?